workflows/GeneSeqToFamily/tutorial.md

```diff
@@ -0,0 +1,86 @@
+# Introduction
+
+This tutorial uses GeneSeqToFamily a Galaxy workflow published by Thanki et al. (2018). The goal of this workflow was to investigate “Discovery and Visualisation of Homologous Genes and Gene Families Using Galaxy”
+
+## GeneSeqToFamily workflow
+
+GeneSeqToFamily workflow can be downloaded from Galaxy ToolShed or from GitHub and then can be inported into local Galaxy or [public instance](usegalaxy.eu).
+
+
+# Importing dataset
+
+### Hands-on: Data upload
+1. Make sure you have an empty analysis history. Give it a name.
+### Tip: Starting a new history
+* Click the gear icon at the top of the history panel
+* Select the option Create New from the menu
+
+
+2. Import Sample Data
+* FASTA file:  [`CDS.fasta`](https://doi.org/10.5281/zenodo.1256760)
+* JSON file: [`gene.json`](https://doi.org/10.5281/zenodo.1256762)
+* Species tree: [`species.nhx`](https://doi.org/10.5281/zenodo.1256753)
+### Tip: Importing data via links
+* Copy the link locations
+* Open the Galaxy Upload Manager
+* Select Paste/Fetch Data
+* Paste the link into the text field
+* Press Start
+### Tip: Change the file type text to nhx once the data file is in your history
+Click on the pencil button displayed in your data file in the history
+* Choose Datatype on the top
+* Select nhx
+* Press save
+
+### 
+	Rename the dataset to “First dataset”
+
+By default, when data is imported via its link, Galaxy names it with its URL.
+
+# Data Preparation
+
+To convert uploaded data into the format acceptable by GeneSeqToFamily workflow:
+
+## GeneSeqToFamily preparation 
+GeneSeqToFamily preparation is an open-source tool that converts genomic information in GFF/JSON format to SQLite format for easy access during the workflow, it also adds species information in Fasta header.
```

Comment:
```suggestion
GeneSeqToFamily preparation is a Galaxy tool that converts genomic information from GFF/JSON format to SQLite format for easy access during the workflow. It can also add species information to the header line of the FASTA sequences.
```